example.json

```diff
@@ -43,6 +48,11 @@ contentObjects.json
 
 articles.json
 "_vanilla": {
+    "_textAlignment": {
+        "_title": "left",
+        "_body": "center",
+        "_instruction": "right",
```

Comment:
Comma needs removing. Same for other `_instruction` instances.

Reply:
Done